### PR TITLE
[PR] Remove Yoast title filtering

### LINKED ISSUE #269: Double site name in titles when Yoast SEO is enabled
I believe Yoast is adjusting the title in a way that conflicts with our setting in the theme. We need to properly disable Yoast SEO's handling of titles entirely.

### PR DESCRIPTION
**Pull request by philcable**
Fixes #269 @jeremyfelt #reviewmerge?
This still allows the Yoast "Force rewrite titles" option to function, which may or may not be desirable.

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt I may have been too quick with this pull request. Removing the filter also disables the per-post title modifications via the snippet editor, but has no effect on the OG or Twitter title meta tags. Should we discuss this further before proceeding?

**Comment by jeremyfelt** [comment]
I browsed through the code around Yoast and `wp_title` a bit.

My guesses:

We should probably use something like `get_the_title()` instead of `spine_get_title()` when outputting the title for sharing reasons. That would solve one of our problems.

And we may be able to remove and add the filter on the fly when we use `spine_get_title()` for the document title.

In `spine_get_title()`, something like this may work:

```
function spine_get_title() {
	$site_part = ' ' . get_option( 'blogname' );
	$global_part = ' | Washington State University';

    remove_filter( 'wp_title', array( WPSEO_Frontend::get_instance(), 'title' ), 15 );
	$view_title = wp_title( '|', false, 'right' );
    if ( defined( 'WPSEO_VERSION' ) ) {
            add_filter( 'wp_title', array( WPSEO_Frontend::get_instance(), 'title' ), 15, 3 );
    }
	$title = $view_title . $site_part . $global_part;

	return apply_filters( 'spine_get_title', $title, $site_part, $global_part, $view_title );
}
```

We should also check how `wp_get_document_title()` (introduced in 4.4) is impacted by all of this. WordPress SEO filters that as well and I have no idea where it's used or if we should be using it.

We can also brainstorm this in person a bit tomorrow.

**Comment by philcable** [comment]
Broke modifications out into a plugin (https://github.com/washingtonstateuniversity/WSUWP-Extended-WordPress-SEO).
@jeremyfelt #reviewmerge?

File: js/wpseo-metabox.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(function($){
```
Comment (jeremyfelt) [inline js/wpseo-metabox.js]: We can take this file out now, right?

**Comment by jeremyfelt** [comment]
Looks good - see the comment about `wpseo-metabox.js`. Otherwise, good to go.

**Comment by jeremyfelt** [comment]
Merging this in now. We can coordinate the release of the theme with these changes with the release and activation of https://github.com/washingtonstateuniversity/WSUWP-Extended-WordPress-SEO on Monday.

**Pull request merged**